src/SDKs/TrafficManager/Management.TrafficManager/Microsoft.Azure.Management.TrafficManager.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.TrafficManager</PackageId>
 		<Description>Microsoft Azure Management TrafficManager Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.TrafficManager</AssemblyName>
-		<Version>2.3.0-preview</Version>
+		<Version>2.4.0-preview</Version>
 		<PackageTags>Microsoft Azure TrafficManager management;TrafficManager;TrafficManager management;</PackageTags>
 		<PackageReleaseNotes/>
```

Comment:
Please add `PackageReleaseNotes` similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/33056293d935f23f00595de23dbd8799ea852efb/src/AzureStack/Management/Subscription/Subscription/Microsoft.AzureStack.Management.Subscription.csproj#L10).

Reply:
Done.